src/cpdag.jl

```diff
@@ -1,5 +1,54 @@
 import Base: iterate, length
 
+"""
+    alt_cpdag(g::DiGraph)
+
+Computes CPDAG from a DAG using a simpler adaption of Chickering's conversion algorithm without ordering all edges
+(equivalent to the PC algorithm with `d`-seperation as independence test, see `pc_oracle`.)
```

Comment:
“separation”

Reply:
This doesn't necessarily has to be part of the PR. Implemented this in the context of another project. The old code is perfectly fine (except maybe
```
for w in inneighbors(g, x)
       label[w => x] == compelled || continue
```
instead of only iterating over the compelled incoming edges of x could lead to worse asymptotic run-time than O(n+m).)

Reply:
Ok, that’s something we can think of later, I am fine with with having this code here of course

Reply:
Ok, sounds like a good plan 👍 thanks for being so open for my contributions :)